src/platform/forms-system/test/js/review/ArrayField.unit.spec.jsx

```diff
@@ -224,6 +224,68 @@ describe('Schemaform review <ArrayField>', () => {
     expect(tree.everySubTree('.schemaform-review-array-warning')).to.not.be
       .empty;
   });
+  it('should render start in edit mode for duplicate items', () => {
+    const idSchema = {};
+    const schema = {
+      type: 'array',
+      items: [
+        {
+          type: 'object',
+          properties: {
+            field: {
+              type: 'string',
+            },
+          },
+        },
+        {
+          type: 'object',
+          properties: {
+            field: {
+              type: 'string',
+            },
+          },
+        },
+      ],
+      additionalItems: {
+        type: 'object',
+        properties: {
+          field: {
+            type: 'string',
+          },
+        },
+      },
+    };
+    const uiSchema = {
+      'ui:title': 'List of things',
+      items: {},
+      'ui:options': {
+        viewField: f => f,
+        itemName: 'Item name',
+        duplicateKey: 'field',
+      },
+    };
+    const arrayData = [{ field: 'a' }, { field: 'b' }, { field: 'a' }];
```

Comment:
Bonus points for changing the case of that duplicate to demonstrate the case insensitivity of the duplicate finder.
```suggestion
    // Duplicates are case insensitive
    const arrayData = [{ field: 'a' }, { field: 'b' }, { field: 'A' }];
```